sky/clouds/gcp.py

```diff
@@ -112,6 +112,66 @@
 # For GPU-related package version, see sky/clouds/service_catalog/images/provisioners/cuda.sh
 _DEFAULT_GPU_IMAGE_ID = 'skypilot:custom-gpu-ubuntu-2204'
 _DEFAULT_GPU_K80_IMAGE_ID = 'skypilot:k80-debian-10'
+# Use COS image with GPU Direct support. Need to contact GCP support to build our own image for GPUDirect-TCPX support.
+# https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/examples/machine-learning/a3-highgpu-8g/README.md#before-starting
+_DEFAULT_GPU_DIRECT_IMAGE_ID = 'projects/cos-cloud/global/images/cos-109-17800-519-1'
```

Comment:
Should we add this to `images.csv` like the GPU image above?

Reply:
Add this in PR https://github.com/skypilot-org/skypilot-catalog/pull/122/files, PTAL. Thanks!